presto-main/etc/certificate-authenticator.properties

```diff
@@ -0,0 +1 @@
+certificate-authenticator.name=standard
```

Comment:
Remove this file, since the configs here are for local development, which is not setup for HTTPS.